src/app/angular/app.component.ts

```diff
@@ -1,11 +1,34 @@
-import { Component, Input } from '@angular/core';
+import { BreakpointObserver, BreakpointState, Breakpoints } from '@angular/cdk/layout';
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { ToastrService } from 'ngx-toastr';
+import { Subject, takeUntil } from 'rxjs';
 
 @Component({
   selector: 'app-component',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
 })
-export class AppComponent {
+export class AppComponent implements OnInit, OnDestroy {
   static selector = 'appComponent';
+  private readonly destroyed: Subject<void> = new Subject<void>();
```

Comment:
Nit: I generally prefer omitting an explicit type when it can be inferred from the initializer value (but I know we think differently on this one) :grin:

Reply:
My opinion comes from Java, where everything, whether the primitive type or not, must have a specific type 😄  I don't insist on it when it comes to TypeScript.

Changed